com.behringer.XAirEdit.yml

```diff
@@ -0,0 +1,40 @@
+app-id: com.behringer.XAirEdit
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+command: X-AIR-Edit
+tags:
+- proprietary
+finish-args:
```

Comment:
How does that work, `--device=all` is not needed to connect(?) to the external device?

Reply:
Ah, it's fully digital. Never mind!